apps/portals/src/configurations/elportal/synapseConfigs/projects.ts

```diff
@@ -49,20 +49,20 @@ export const projectsDetailsPageConfiguration: DetailsPageProps = {
   synapseConfigArray: [
     {
       name: 'CardContainerLogic',
-      columnName: 'grantNumber',
+      columnName: 'grant',
       title: 'Studies',
-      tableSqlKeys: ['grantNumber'],
+      tableSqlKeys: ['grant'],
```

Comment:
The `studies` cards shown on the details page should be filtered using the `grant` column